src/parser/warrior/protection/modules/features/BlockCheck.js

```diff
@@ -85,4 +118,4 @@ class ShieldBlockMitigation extends Analyzer {
   statisticOrder = STATISTIC_ORDER.CORE(10);
 }
 
-export default ShieldBlockMitigation;
\ No newline at end of file
+export default BlockCheck;
```

Comment:
fun fact: you can write `export default class BlockCheck extends Analyzer { .... }` and not have to duplicate the class name